internal/bus/topics.go

```diff
@@ -6,19 +6,21 @@
 package bus
 
 const (
-	AddInstancesTopic          = "add-instances"
-	UpdatedInstancesTopic      = "updated-instances"
-	DeletedInstancesTopic      = "deleted-instances"
-	ResourceUpdateTopic        = "resource-update"
-	NginxConfigUpdateTopic     = "nginx-config-update"
-	InstanceHealthTopic        = "instance-health"
-	ConfigUploadRequestTopic   = "config-upload-request"
-	DataPlaneResponseTopic     = "data-plane-response"
-	ConnectionCreatedTopic     = "connection-created"
-	ConfigApplyRequestTopic    = "config-apply-request"
-	WriteConfigSuccessfulTopic = "write-config-successful"
-	ConfigApplySuccessfulTopic = "config-apply-successful"
-	ConfigApplyFailedTopic     = "config-apply-failed"
-	RollbackCompleteTopic      = "rollback-complete"
-	RollbackWriteTopic         = "rollback-write"
+	AddInstancesTopic           = "add-instances"
+	UpdatedInstancesTopic       = "updated-instances"
+	DeletedInstancesTopic       = "deleted-instances"
+	ResourceUpdateTopic         = "resource-update"
+	NginxConfigUpdateTopic      = "nginx-config-update"
+	InstanceHealthTopic         = "instance-health"
+	ConfigUploadRequestTopic    = "config-upload-request"
+	DataPlaneResponseTopic      = "data-plane-response"
+	ConnectionCreatedTopic      = "connection-created"
+	ConfigApplyRequestTopic     = "config-apply-request"
+	WriteConfigSuccessfulTopic  = "write-config-successful"
+	ConfigApplySuccessfulTopic  = "config-apply-successful"
+	ConfigApplyFailedTopic      = "config-apply-failed"
+	RollbackCompleteTopic       = "rollback-complete"
+	RollbackWriteTopic          = "rollback-write"
+	DataplaneHealthTopic        = "dataplane-health"
```

Comment:
Hey, wondering why we need two topics for the data plane health topic ?

Reply:
Hey, we are using
`DataplaneHealthTopic`: This topic is published by the `command_plugin` whenever a health request is received. The `watcher_plugin` subscribes to it and triggers the next stage by publishing the DataplaneHealthProcessTopic with the instances health data.
`DataplaneHealthProcessTopic`: This topic is subscribed to by the command_plugin, which then initiates the health RPC i.e sending the health status of instances to mpi and finally sends the data response to the mpi.   
we need both topics because they handle different parts of the health request workflow.

Reply:
Ok thank you that makes sense, didn't understand the process topic.